### ISSUE TITLE
Stop location update after location found

### ISSUE THREAD

**Issue by neoacevedo** [domoritz/leaflet-locatecontrol#230]
I have the following issue:

I draw a marker and get the current location on load page. 
I drag/drop the marker to a desired location.
I press the location control button to turn back to the current location.

Now I need to stop this control to update the location:

```js
function onLocationFound(e) {
    var radius = e.accuracy / 2;
    $.get('https://nominatim.openstreetmap.org/reverse', {
        format: "json",
        addressdetails: 0,
        lat: e.latlng.lat,
        lon: e.latlng.lng
    },
            function (data) {
                $("#address").val(data.display_name);
            });

    if(typeof (marker) != "undefined") {
        marker.removeFrom(map);
        circle.removeFrom(map);
    }
    marker = L.marker(e.latlng, {draggable: true});
    marker.addTo(map)
            .bindPopup("You are within " + radius + " meters from this point").openPopup();
    marker.on('drag', onDrag);

    circle = L.circle(e.latlng, radius);
    circle.addTo(map);
    map.setView(e.latlng, 16);
    lc.stopFollowing(); // <----- I tried this but simply doesn't work.
}
```
How do I achieve it?

**Comment by domoritz** [comment]
It should work. Can you step into the code with the debugger and see why it does not work?

**Comment by neoacevedo** [comment]
lc.stopFollowing is not a function.

If I simply call lc.stop(), works.

**Comment by domoritz** [comment]
It exists, though: https://github.com/domoritz/leaflet-locatecontrol/blob/3de0f43358a4a943d1cdebc24011c9d785a55382/src/L.Control.Locate.js#L443

**Comment by neoacevedo** [comment]
In the minified version it doesn't.

**Comment by domoritz** [comment]
Hmm, odd.

**Comment by domoritz** [comment]
Thanks for the issue. Should be fixed in v0.65.2
